test/integration/multitenant/multitenant_test.go

```diff
@@ -110,6 +110,10 @@ func TestMultitenant(t *testing.T) {
 					clusterName := regexp.MustCompile(`\/clusters\/([^\/]*)$`).FindStringSubmatch(membershipOp.Get("endpoint.gkeCluster.resourceLink").String())[1]
 					clusterOp := gcloud.Runf(t, "container clusters describe %s --location %s --project %s", clusterName, clusterLocation, clusterProjectID)
 
+					// Extract enablePrivateEndpoint flag value
+					enablePrivateEndpoint := clusterOp.Get("privateClusterConfig.enablePrivateEndpoint").Bool()
+					assert.True(enablePrivateEndpoint, "The cluster external endpoint must be disabled.")
```

Comment:
super nit: but "private" is probably more accurate than "disabled"

```suggestion
					assert.True(enablePrivateEndpoint, "The cluster external endpoint must be private.")
```